### ISSUE TITLE
ManifoldLearning

### ISSUE THREAD

**Issue by mateuszbaran** [JuliaManifolds/ManifoldML.jl#4]
There is a package ManifoldLearning: https://github.com/wildart/ManifoldLearning.jl . Only partially related but we may still want to take a look at what it does.

**Comment by kellertuer** [comment]
They try to estimate the low dimensional manifold the data (from some high-dimensional vector space) lies on and provide properties of that; so the setting is different.

**Comment by mateuszbaran** [comment]
Yes but we could have a similar thing just for high-dimensional manifolds. Would that be manifold manifold learning? :slightly_smiling_face: For example laplacian eigenmaps could be easily adapted, probably some other too, by replacing metric in the kNN step.

**Comment by kellertuer** [comment]
That would then be Laplace Beltrami eigenmaps? I just have no experience in this direction.

**Comment by mateuszbaran** [comment]
There seems to be a lot of slightly different variations on the same idea. Do you have a reference for Laplace Beltrami eigenmaps? I only have the basic idea of how such embeddings work.

**Comment by kellertuer** [comment]
No, I am not aware of a (good) reference, because I haven't worked much with Laplace Beltrami per se.
